test/honeybadger/json_test.exs

```diff
@@ -23,19 +23,11 @@ defmodule Honeybadger.JSONTest do
     test "encodes notice when context has structs" do
       error = %RuntimeError{message: "oops"}
       struct = %Request{ip: "0.0.0.0"}
-      map = Map.from_struct(struct)
 
-      {:ok, custom_encoded} =
-        error
-        |> Notice.new(%{context: %{a: struct, b: [struct], c: {struct, struct}}}, [])
-        |> JSON.encode()
-
-      {:ok, jason_encoded} =
-        error
-        |> Notice.new(%{context: %{a: map, b: [map], c: [map, map]}}, [])
-        |> Jason.encode()
-
-      assert custom_encoded == jason_encoded
+      assert {:ok, _json} =
```

Comment:
Has the behavior changed here? I thought it was testing that map and struct ends up in the same json being generated.

Reply:
The behavior changed in OTP 26 because the map order wasn't guaranteed. The test was asserting more than safely encoding structs, and I simplified it.

Since you pointed it out, I've pushed an alternate approach that retains the original test.

Reply:
Gotcha, thanks!